linode/profile/datasource_test.go

```diff
@@ -29,15 +29,12 @@ func TestAccDataSourceProfile_basic(t *testing.T) {
 					resource.TestCheckResourceAttrSet(resourceName, "email_notifications"),
 					resource.TestCheckResourceAttrSet(resourceName, "ip_whitelist_enabled"),
 					resource.TestCheckResourceAttrSet(resourceName, "lish_auth_method"),
-					resource.TestCheckResourceAttr(resourceName, "authorized_keys.#", "0"),
```

Comment:
Getting rid of assertion here and some below as it causes the test to fail depending on which test account. Test coverage for these attributes are only available in [unit test](https://github.com/linode/terraform-provider-linode/pull/1429/files#diff-78e72c6589b04481a8297b75ebcc5d5b5f4cfd4b40bed49ef620f2761742cc26) for now.